scripts/commands/open.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------------------------------------------------------------
+-- func: open <time>
+-- desc: Forces a door to open
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 2,
+    parameters = "i"
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!open <time>")
+end
+
+function onTrigger(player, open_time, target)
+    -- validate target
+    local targ = player:getCursorTarget()
+    if targ == nil then
+        error(player, "A door NPC must be selected.")
+        return
+    end
+
+    targ:openDoor(open_time)
+
+
```

Comment:
Extra whitespace

Reply:
That is how they had the command. I have tested the command and got no errors

Reply:
Who's _they_?

Reply:
Twilight on Wings

Reply:
Gonna reply to this in the main thread